modules/nw-configuring-lb-allowed-source-ranges.adoc

```diff
@@ -0,0 +1,30 @@
+// Modules included in the following assemblies:
+//
+// * networking/configuring_ingress_cluster_traffic/configuring-ingress-cluster-traffic-load-balancer-allowed-source-ranges.adoc
+
+:_content-type: PROCEDURE
+[id="nw-configuring-lb-allowed-source-ranges_{context}"]
+= Configuring Load Balancer allowed source ranges
+
+By configuring Load Balancer allowed source ranges, you can limit the access to the Load Balancer for the `ingresscontroller` to a specified list of IP address ranges. The Ingress Operator reconciles the Load Balancer Service and sets the `.Spec.LoadBalancerSourceRanges` based on `AllowedSourceRanges`.
+
+[NOTE]
+====
+If you have already set the `.Spec.LoadBalancerSourceRanges` field or the Load Balancer service anotation `service.beta.kubernetes.io/load-balancer-source-ranges` in a previous version of {product-title}, Ingress Controller starts reporting `Progressing=True` after an upgrade. To fix this, set `AllowedSourceRanges` that overwrites `.Spec.LoadBalancerSourceRanges` field and clears `service.beta.kubernetes.io/load-balancer-source-ranges` annotation. Ingress Controller starts reporting `Progressing=False` again.
+====
+
+.Prerequisites
+
+* You need a deployed Ingress Controller on a running cluster.
```

Comment:
I'd consider "You must have" or "You have a"

Reply:
addressed